projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -181,6 +181,8 @@ export class TuiInputPhoneComponent
         if (this.nativeValue === this.nonRemovablePrefix || this.isTextValue) {
```

Comment:
Should it be `this.nativeValue.trim() === this.nonRemovablePrefix.trim()`? Is it possible one of them contains space at the end and the other does not?

Reply:
>  Is it possible one of them contains space at the end and the other does not?

* `nonRemovablePrefix` always includes space at the end:
https://github.com/taiga-family/taiga-ui/blob/a67423fe82616925ebaeb912b627b9e66a6e3984/projects/kit/components/input-phone/input-phone.component.ts#L227-L229

* User can't delete space in `nativeValue`
https://github.com/taiga-family/taiga-ui/blob/a67423fe82616925ebaeb912b627b9e66a6e3984/projects/kit/components/input-phone/input-phone.component.ts#L269